custom/icds_reports/management/commands/queue_person_cases.py

```diff
@@ -0,0 +1,29 @@
+
+from django.core.management.base import BaseCommand
+from corehq.apps.userreports.tasks import _get_config_by_id
+from corehq.apps.userreports.util import get_table_name
+from corehq.apps.userreports.models import AsyncIndicator
+from django.db import connections
+
+from dimagi.utils.chunked import chunked
+
+
+class Command(BaseCommand):
+    help = "Rebuild Bihar person cases "
+
+    def handle(self):
+        person_config = _get_config_by_id('static-icds-cas-static-person_cases_v3')
+        table_name = get_table_name('icds-cas', 'static-person_cases_v3')
+        query = """select doc_id from "{}" where state_id='f9b47ea2ee2d8a02acddeeb491d3e175' order by supervisor_id, doc_id""".format(table_name)
```

Comment:
To resume, you will need to fetch all the ordering columns

Reply:
why, I mean. Wont the ordering working like this? Basically if I just order by doc_id, that would also work no?

Reply:
but you are ordering by `supervisor_id, doc_id`. so just having the doc_id is not enough info to resume. if you just order by doc_id it will be, but I suspect that will be much slower since we don't have an ordered index on doc_id (although i can't get good explains, they seem to ignore the sort part)